src/hook/usePDF.tsx

```diff
@@ -47,16 +47,16 @@ const usePDF = ({
 	viewer
 }: IUsePDF): TUsePDF => {
 	const [pdfDoc, setPdfDoc] = useState<PDFDocumentProxy>();
-	const [pages, setPages] = useState<(JSX.Element | undefined)[]>([]);
+	const [pages, setPages] = useState<(ReactElement | undefined)[]>([]);
```

Comment:
Set the type to: `ReactElement<IPDFPage>`, this then means you don't need to do `as IPDFPage;` on lines 224 and 297